src/sql/workbench/contrib/notebook/browser/cellViews/code.component.ts

```diff
@@ -277,6 +276,41 @@ export class CodeComponent extends CellView implements OnInit, OnChanges {
 			DOM.getContentWidth(this.codeElement.nativeElement),
 			DOM.getContentHeight(this.codeElement.nativeElement)));
 		this._editor.setHeightToScrollHeight(false, this._cellModel.isCollapsed);
+		this.horizontalScrollbar();
+	}
+
+	public horizontalScrollbar(): void {
+		if (this._configurationService.getValue('editor.wordWrap') === 'off' && this.cellModel.cellType !== CellTypes.Code) {
+			// Get markdown split view horizontal scrollbar
+			let horizontalScrollbar = this.codeElement.nativeElement.querySelector('div.invisible.scrollbar.horizontal');
+
+			// Compare the editor bottom position to the scrollable area in order to decide whether to make the horizontal scrollbar fixed or absolute
+			let editorBottomPos = this.codeElement.nativeElement.closest('.show-markdown .editor').getBoundingClientRect().bottom - this.codeElement.nativeElement.closest('.show-markdown .editor').getBoundingClientRect().top;
+			let viewportHeight = DOM.getTotalHeight(document.querySelector('.scrollable'));
+
+			// Horizontal scrollbar values used when fixed
+			let viewportTop = document.querySelector('.scrollable').getBoundingClientRect().top;
+			let horizontalTop = viewportTop + viewportHeight - horizontalScrollbar.scrollHeight - 10;
```

Comment:
I'm not excited about the hard-coded 10px value here. Can you get that value instead from the height of the element that pushes this horizontal top down?